.travis.yml

```diff
@@ -0,0 +1,47 @@
+#Travis CI configuration for KWIVER
+# See http://travis-ci.org/Kitware/kwiver/
+
+sudo: false
+
+language: cpp
+compiler:
+  - gcc
+  - clang
+
+cache:
+  directories:
+  - $HOME/deps
+
+before_script:
+  - bash .travis/install-deps.sh
+
+addons:
+  apt:
+    packages:
+    - libproj-dev
+    - libgl1-mesa-dev
+    - libxt-dev
+    - libqt4-dev
+    - libatlas-base-dev
+
+script:
+  - export PATH=$HOME/deps/bin:$PATH
+  - mkdir build
+  - cd build
+  - cmake -DKWIVER_ENABLE_ARROWS=ON
```

Comment:
Yes this will need KWIVER_USE_BUILD_TREE=ON once we enable the sprokit build.  For now it works fine and the Travis CI build is passing.  I suggest we merge it since the test passes and enable that option when we need it.

Better yet, does it make sense to just default KWIVER_USE_BUILD_TREE to ON?  Or maybe default to ON when tests are enabled?  It's kind of odd that the tests don't pass unless you manually enable this option.  Alternatively, Maybe there is a way to explicitly insert the build tree paths when the test code is being run?